SETUP_SUMMARY_PRD.md

```diff
@@ -0,0 +1,32 @@
+
```

Comment:
I would move this to doc folder and maybe put a ref to it somewhere in the generale README